<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Local Settings</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Local settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Local peer name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="peerName"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Number of active threads</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="threadCount">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="computeThreadCount">
        <property name="text">
         <string>Compute number of active threads</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Local key</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Fingerprint:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="keyLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="generateKey">
        <property name="text">
         <string>Generate new key...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="importKey">
        <property name="text">
         <string>Import key...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Network settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Bootstrapping:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Endpoints:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="useCompression">
        <property name="text">
         <string>Use compression for outgoing source data</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SettingsDialog</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>onAccepted()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>computeThreadCount</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>computeThreadCount()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>generateKey</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>generateKey()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>importKey</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>importKey()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>generateKey()</slot>
  <slot>importKey()</slot>
  <slot>onAccepted()</slot>
  <slot>computeThreadCount()</slot>
 </slots>
</ui>
